### PR TITLE
Add kubernetes example yamls

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: k8s/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+~ kubectl create -f warp-job.yaml
+```
+
+```
+~ kubectl get pods -l job-name=warp-job
+NAME             READY   STATUS      RESTARTS   AGE
+warp-job-6xt5k   0/1     Completed   0          8m53s
+```
+
+To obtain the console output look at the job logs
```
Comment (klauspost) [inline k8s/README.md]: Is it possible to get the file with the run statistics?

Comment (harshavardhana) [reply k8s/README.md]: Yes I need to add local persistence to get the output, but I think in such situations it is better to dump this output back to object storage?

**Comment by nitisht** [comment]
Should we add the helm chart also in this repo. https://github.com/nitisht/charts/commit/fed5a9768df5193a1f0e5879e17fc3b3debf2a0c

There is no need for it to be hosted on the central helm repo anyways

**Comment by harshavardhana** [comment]
> Should we add the helm chart also in this repo. [nitisht/charts@fed5a97](https://github.com/nitisht/charts/commit/fed5a9768df5193a1f0e5879e17fc3b3debf2a0c)
> 
> There is no need for it to be hosted on the central helm repo anyways

Most definitely send a PR.

**Pull request merged**